style.css

```diff
@@ -0,0 +1,3 @@
+h2 {
```

Comment:
**Optional**

I will suggest that you use CSS selectors like 'Class' or 'Id' to identify various HTML elements.
The main benefit of setting class or ID is that you can present the same HTML element differently, depending on its class or ID.